internal/worker/server.go

```diff
@@ -133,7 +133,7 @@ func (s *Server) EnqueueManifestJobByID(job *ManifestJobByID, parent uuid.UUID,
 }
 
 func (s *Server) enqueue(jobType string, job interface{}, dependencies []uuid.UUID, channel string) (uuid.UUID, error) {
-	prometheus.EnqueueJobMetrics(jobType)
+	prometheus.EnqueueJobMetrics(jobType, channel)
```

Comment:
Here we could call the function like this:
`prometheus.EnqueueJobMetrics(jobType, channel[4:])`